cg/cli/demultiplex/sample_sheet.py

```diff
@@ -102,6 +101,10 @@ def create_sheet(
     )
 
 
+def _list_to_csv_stream(nested_list: List[List[str]]) -> str:
+    return WriteStream().write_stream_from_content(file_format=FileFormat.CSV, content=nested_list)
```

Comment:
```suggestion
    return WriteStream.write_stream_from_content(file_format=FileFormat.CSV, content=nested_list)
```
Do we need this extra utility function? I think you can call Writestream directly in the click echo

Reply:
Reasonable, changed now.